src/main/java/core/basesyntax/controller/ControllerHandler.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ControllerHandler {
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String IGNORE_CASE = "exit";
```

Comment:
```suggestion
    private static final String EXIT = "exit";
```

let's make scanner a local variable